konfera/models/ticket.py

```diff
@@ -2,6 +2,8 @@
 from django.utils.translation import ugettext_lazy as _
 
 from konfera.models.speaker import TITLE_UNSET, TITLE_CHOICES
+from konfera.models.order import Order
+from django.utils import timezone
```

Comment:
please move django import together, and leave one blank line between django and konfera imports, so it is according PEP8.

https://www.python.org/dev/peps/pep-0008/#imports